package.json

```
@@ -8,4 +12,4 @@
     "rollup": "^2.67.0",
     "uvu": "^0.5.3"
   }
-}
+}
```

```suggestion
}

```



please keep newlines

if your text editor supports editorconfig, this shouldn't be a problem in the future